rest/src/main/java/com/netflix/conductor/rest/controllers/TaskResource.java

```diff
@@ -111,6 +111,17 @@ public Map<String, Integer> size(
         return taskService.getTaskQueueSizes(taskTypes);
     }
 
+    @GetMapping("/queue/size")
```

Comment:
Can the API above in line 107 - ```@GetMapping("/queue/sizes")``` be modified to support this?

Reply:
If i were to modify it, i would make it like the new api that's added. `/queue/sizes` accepts a list and is unbounded. We can deprecate `/queue/sizes` and remove it in an upcoming release.